.github/workflows/main.yml

```
@@ -116,65 +118,66 @@ jobs:
                 sudo mv "$BACKEND_OLD_PATH" "$BACKEND_PATH"
                 echo "Backend restaurado a versión anterior."
               fi
               sudo systemctl restart kioscomanager.service || true
               echo "Rollback de Backend completado."
             }
-            trap rollback_backend ERR # Si algo falla en este bloque, se ejecuta el rollback
+            trap rollback_backend ERR
 
-            sudo systemctl stop kioscomanager.service || true # Detener servicio (ignorar error si ya está detenido)
+            sudo systemctl stop kioscomanager.service || true
             
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)